common/handlers/groups/patch.go

```diff
@@ -0,0 +1,116 @@
+package groups
+
+import (
+	"context"
+
+	dsc "github.com/aserto-dev/go-directory/aserto/directory/common/v3"
+	dsr "github.com/aserto-dev/go-directory/aserto/directory/reader/v3"
+	dsw "github.com/aserto-dev/go-directory/aserto/directory/writer/v3"
+	"github.com/aserto-dev/scim/common"
+	"github.com/aserto-dev/scim/common/convert"
+	"github.com/elimity-com/scim"
+	serrors "github.com/elimity-com/scim/errors"
+	"github.com/rs/zerolog"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	structpb "google.golang.org/protobuf/types/known/structpb"
+)
+
+func (g GroupResourceHandler) Patch(ctx context.Context, id string, operations []scim.PatchOperation) (scim.Resource, error) {
+	logger := g.logger.With().Str("method", "Patch").Str("id", id).Logger()
+	logger.Info().Msg("patch group")
+
+	if !g.cfg.HasGroups() {
+		logger.Error().Msg("groups not enabled")
+		return scim.Resource{}, serrors.ScimErrorBadRequest("groups not enabled")
+	}
+
+	getObjResp, err := g.dirClient.DS().Reader.GetObject(ctx, &dsr.GetObjectRequest{
+		ObjectType:    g.cfg.Group.SourceObjectType,
+		ObjectId:      id,
+		WithRelations: false,
+	})
+	if err != nil {
+		logger.Error().Err(err).Msg("failed to get group")
+		st, ok := status.FromError(err)
+
```

Comment:
The spacing here is strange. The newline should come after `logger.Error()...`, not after the `FromError(err)` calls.